src/Compiler.jl

```diff
@@ -3209,8 +3217,9 @@ Generate Julia code to call the XLA executable.
 
 - `flatten_names`: A list of `Symbol`s representing the names of the flattened linear arguments.
 - `nresults`: The number of results to expect.
+- `is_pure`: Whether the function being compiled is pure (e.g. has a side effect, like MPI.Send)
```

Comment:
```suggestion
- `is_pure`: Whether the function being compiled is pure (i.e., has no side effects)
```
a little confusing as is